### PR TITLE
Fix issues with LACP and nexthop LAG action

### PR DESCRIPTION
**Pull request by n-sandeep**
Issues fixed are:

- Populate tx_lag_table, modify exact to ternary match for this table.
- Populate RIF tables
- Populate correct dmac_high and dmac_low fields correctly
- Handle LACP bond interface delete and re-add

### CODE REVIEW

File: switchapi/es2k/lnw_v3/switch_pd_lag.c [hunk 1]
```diff
@@ -190,7 +190,7 @@ switch_status_t switch_pd_tx_lag_table_entry(switch_device_t device,
     }
 
     status = tdi_key_field_set_value_and_mask(key_hdl, field_id_hash,
-                                              lag_list + port_count, 0x3);
+                                              lag_list + port_count, 0x7);
```
Comment (5abeel) [inline switchapi/es2k/lnw_v3/switch_pd_lag.c:193]: what is the `0x7` here?

**Review by 5abeel — APPROVED** [review]
LGTM

File: switchlink/switchlink_db.c [hunk 2]
```diff
@@ -1132,6 +1132,31 @@ switchlink_db_status_t switchlink_db_delete_lag_member(
...
+ *    LAG member ifindex on success
+ *    -1 otherwise
+ */
+uint32_t switchlink_db_delete_lacp_member(switchlink_handle_t lag_h) {
+  tommy_node* node = tommy_list_head(&switchlink_db_lag_member_obj_list);
+  while (node) {
+    switchlink_db_lag_member_obj_t* obj = node->data;
+    krnlmon_assert(obj != NULL);
+    node = node->next;
+    if ((obj->lag_member_info.lag_h == lag_h)) {
```
Comment (ffoulkes) [inline switchlink/switchlink_db.c:1153]:
```suggestion
    if (obj->lag_member_info.lag_h == lag_h) {
```

**Review by ffoulkes — APPROVED** [review]
LGTM

**Pull request merged**